exporter/SynthesisFusionAddin/src/Parser/SynthesisParser/Joints.py

```diff
@@ -402,7 +402,8 @@ def _jointOrigin(
     geometryOrOrigin = (
         (
             fusionJoint.geometryOrOriginOne
-            if fusionJoint.geometryOrOriginOne.objectType == "adsk::fusion::JointGeometry"
+            if fusionJoint.geometryOrOriginOne.objectType
+            == "adsk::fusion::JointGeometry"
```

Comment:
I don't really like this formatting. Has this been the config for a while and we just haven't run the formatter? It seems to really want to split things into multiple lines that end up making it more verbose and harder to read. Later on it split an if with a single variable flag into 3 lines with parentheses because there was a comment. Is the line width too low?

Reply:
Actually I just checked #985 where the formatting was fixed so this isn't an issue.